src/bun.js/modules/NodeModuleModule.cpp

```diff
@@ -628,6 +632,75 @@ static JSValue getGlobalPathsObject(VM& vm, JSObject* moduleObject)
         static_cast<ArrayAllocationProfile*>(nullptr), 0);
 }
 
+JSC_DEFINE_HOST_FUNCTION(jsFunctionSetCJSWrapperItem, (JSGlobalObject * globalObject, JSC::CallFrame* callFrame))
+{
+    auto scope = DECLARE_THROW_SCOPE(globalObject->vm());
+    JSValue a = callFrame->argument(0);
+    JSValue b = callFrame->argument(1);
+    Zig::GlobalObject* global = jsDynamicCast<Zig::GlobalObject*>(globalObject);
+    if (!global) return JSC::JSValue::encode(JSC::jsUndefined());
```

Comment:
why is `!global` checked?

Reply:
the cast can fail but since i wrote this i found the function im supposed to use